plugins/algoltek-usb/fu-algoltek-usb-device.c

```diff
@@ -74,6 +74,8 @@ fu_algoltek_usb_device_rdr(FuAlgoltekUsbDevice *self, int address, GError **erro
 static GByteArray *
 fu_algoltek_usb_device_rdv(FuAlgoltekUsbDevice *self, GError **error)
 {
+	guint8 underscore_count = 0;
```

Comment:
I think the `guint8 underscore_count = 0;` can go inside the `if (` -- no?

Reply:
Updated!